docs/bnb_smart_chain/staking/developer-guide.md

```diff
@@ -0,0 +1,247 @@
+# BSC Staking Developer Guide
```

Comment:
The title of the sidebar is `Developer Staking Guide`, not good title.
what about `Build Staking dApps Guide`.

Reply:
fixed